atlas/rules/semibin.smk

```diff
@@ -57,7 +57,7 @@ rule semibin_generate_data_multi:
         fasta=rules.combine_contigs.output,
         bams=expand(rules.sort_bam.output, sample=SAMPLES),
     output:
-        expand("Cobinning/SemiBin/{sample}/{files}",
```

Comment:
@SilasK  While I run this pipeline, semibin create a folders called "samples", and {sample}/{files} is under this folder. the previous output path seem not to find csv files.

Reply:
I don't know why there is a different behavior on my and your computer. I will test it again on a different one.